### ISSUE TITLE
Can't get translations for locale terms at any language.

### ISSUE THREAD

**Issue by PrimoQA** [michel-kraemer/citeproc-java#93]
Hello Michel, thank you for your help.
We can’t manage to get translations for locale terms in any language, the conjunctions are left not translated.
In order to set the language we call the CSLItemDataBuilder.language(String language) method, with 5 letters language for example “it-IT”.

For an example, for an Italian record, this was the citation outcome:
	Ferrari, Oreste., Papaldo, Serenita, **and** Papaldo, S. 1999. Le sculture del Seicento a Roma. Roma: Ugo Bozzi.

The term exists in the locale file but we still can't get the translation:
	term name="and">e</term

Those the CSL object fields:
	 {"id":"-GEN-11nf9bmqi9","type":"book",**"language":"it-IT"**,"author":[{"family":"Ferrari","given":"Oreste."},{"family":"Papaldo, 	Serenita","given":""},{"family":"Papaldo, S.","given":""}],"issued":{"date-parts":[[1999]]},"container-title":"","DOI":"","edition":"","ISBN":"887003030X","ISSN":"","issue":"","keyword":"Sculpture, Italian -- Italy -- Rome -- 17th century -- Catalogs; Church decoration and ornament -- Italy -- Rome -- Catalogs; Sculpture -- Italy -- Rome -- Catalogs; Sculpture -- Vatican City -- Catalogs","note":"","number-of-pages":"","page":"","page-first":"","PMID":"","publisher":"Ugo Bozzi","publisher-place":"Roma","section":"","title":"Le sculture del Seicento a Roma","URL":"","volume":""}

Thanks

**Comment by michel-kraemer** [comment]
Which version of citeproc-java are you using? Is it possible to upload example code that demonstrates the issue so I can reproduce it? Thanks.

**Comment by michel-kraemer** [comment]
I tried to reproduce the issue with the following code:

```java
CSLItemData item = new CSLItemDataBuilder()
    .type(CSLType.BOOK)
    .language("it-IT")
    .author(new CSLNameBuilder().family("Ferrari").given("Oreste.").build(),
            new CSLNameBuilder().family("Papaldo, Serenita").build(),
            new CSLNameBuilder().family("Papaldo, S.").build()
    )
    .issued(new CSLDateBuilder().dateParts(1999).build())
    .publisher("Ugo Bozzi")
    .publisherPlace("Roma")
    .title("Le sculture del Seicento a Roma")
    .build();

CSL csl = new CSLBuilder()
    .style("ieee")
    .lang("it-IT")
    .itemDataProvider(new ListItemDataProvider(item))
    .build();
csl.setOutputFormat("text");
csl.registerCitationItems(item.getId());
System.out.println(csl.makeBibliography().makeString());
```

The output is:

```
[1]O. Ferrari, Papaldo, Serenita, e Papaldo, S., Le sculture del Seicento a Roma. Roma: Ugo Bozzi, 1999.
```

So, it seems the locale is used correctly. I'm using citeproc-java 3.0.0-SNAPSHOT.

Please let me know if my code covers your issue or what you are doing instead. Thanks.

**Comment by PrimoQA** [comment]
Hello Michel, thank you very much for the detailed answer.
I see that you used CSLBuilder().lang("it-IT"), but we used only CSLItemDataBuilder().language("it-IT").
We also use the CSL.makeAdhocBibliography method.
Is it the reason for the difference?

Those are the dependencies we're using:
    <groupId>org.citationstyles</groupId>
    <artifactId>styles</artifactId>
    <version>20.11-SNAPSHOT</version>
...
    <groupId>org.citationstyles</groupId>
    <artifactId>locales</artifactId>
    <version>20.11-SNAPSHOT</version>
...
    <groupId>de.undercouch</groupId>
    <artifactId>citeproc-java</artifactId>
    <version>2.0.0</version>

Thanks

**Comment by michel-kraemer** [comment]
@PrimoQA As I said in my previous answer, I'm using the latest snapshot 3.0.0-SNAPSHOT. I've just released an alpha version 3.0.0-alpha.1 based on this snapshot. Maybe you can try it again with this version?
